server/src/modules/filestore/file-store-cloud.service.ts

```diff
@@ -49,4 +49,23 @@ export class FileStoreCloudService {
 
         this.logger.log(`DELETE SUCCESS! Deleted file ${fileKey} from bucket ${this.config.get('storage.bucketName')}`);
     }
+
+    async getFileCloud(fileKey: string): Promise<Uint8Array> {
+        try {
+            const cmdRes = await this.s3Client.send(
+                new GetObjectCommand({
+                    Bucket: this.config.get('storage.bucketName'),
+                    Key: fileKey
+                })
+            );
+
+            return cmdRes.Body.transformToByteArray();
```

Comment:
Best method (https://docs.nestjs.com/techniques/streaming-files) seems to be to return this as Nest's StreamablePipe which handles piping for us. So
```ts
async getFileCloud(fileKey: string): Promise<StreamableFile> {
    try {
        const cmdRes = await this.s3Client.send(
            new GetObjectCommand({
                Bucket: this.config.get('storage.bucketName'),
                Key: fileKey
            })
        );

        return new StreamableFile(await cmdRes.Body.transformToByteArray());
```